stories/Header.stories.tsx

```diff
@@ -12,6 +12,11 @@ const meta: Meta = {
     readerState: {},
   },
   argTypes: {
+    currentColorMode: {
+      options: ['day', 'sepia', 'night'],
+      control: { type: 'radio' },
+      description: 'Select between day / sepia / night mode',
+    },
```

Comment:
I wonder if we should make this a global argTypes so that we don't need to redefine them every time we create a new story?

Reply:
Yea seems smart to me

Reply:
Updated.